pages/blog-1.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <link rel="stylesheet" href="../styles/theme.css">
+    <link rel="stylesheet" href="../styles/blog.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Anton+SC&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+        integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Travel Blog</title>
+
+</head>
+
+<!-- body -->
+
+<body>
+
+    <!-- header -->
+    <header>
+        <!-- nav bar-->
+        <nav class="navbar navbar-light navbar-expand-sm justify-content-center" style="background-color: #283a2c;">
+            <div class="navbar-nav">
+                <a class="nav-item nav-link mx-2" href="/index.html"> Home </a>
+                <a class="nav-item nav-link mx-2" href="#"> Photos </a>
+                <a class="nav-item nav-link mx-2 active" href="#"> Blog </a>
+                <a class="nav-item nav-link mx-2" href="#"> Social Media </a>
+                <a class="nav-item nav-link mx-2" href="#"> Countries </a>
+                <a class="nav-item nav-link mx-2" href="#"> Contact us </a>
+            </div>
+        </nav>
```

Comment:
Once merged/now if you want to, we need to update this to match the new nav bar made by holly so it is still mobile accessible!